app/models/concerns/media_information.rb

```diff
@@ -6,60 +6,39 @@ module MediaInformation
   def set_information
```

Comment:
is `MediaInformation` still needed? maybe we can consider merging it back inside `ProjectMedia`. Also, `information` is not a suitable name for what this thing is, because it's too generic a name. What is this thing that we call `information`?